script.js

```
@@ -108,12 +112,14 @@ const r = () => {
         scroll();
     });
 
-    fetch(json).then((res) => {
-        res.json().then((data) => {
-            comments = data.comments;
-            renderChat();
+    if (hasJSON) {
+        fetch(json).then((res) => {
+            res.json().then((data) => {
+                comments = data.comments;
```

nit: I'd say ideally throw an error if comments is undefined.

---

This is Evan's existing code, I just wrapped it in the `if (hasJSON)`

---

Fair enough.